core/src/main/resources/mapfish-spring-custom-fonts.xml

```diff
@@ -10,7 +10,9 @@
                <!-- Custom fonts can be listed here.
                E.g. a font in `/WEB-INF/classes/fonts/MyFont.ttf` can be referenced with `fonts/MyFont.ttf`.
                 -->
-                <!-- <value>fonts/MyFont.ttf</value> -->
+                <value>fonts/FontAwesome5FreeRegular.ttf</value>
+                <value>fonts/FontAwesome5FreeBold.ttf</value>
+                <value>fonts/FontAwesome5Brands.ttf</value>
```

Comment:
Do you test without this change?

Reply:
Well, it passes, but it will not load anything, no ?

I've added a second commit with an example.
On the current state (commented  FA lines in `core/src/main/resources/mapfish-spring-custom-fonts.xml` and FA texts in `examples/src/test/resources/examples/fontawesome/report.jrxml`) I can print but FontAwesome is not used.

Reply:
I think I have to take over...

Reply:
Ok, I let you the hand.